frontend/src/components/Navbar/GlobalNavbar/GlobalNavbar.tsx

```diff
@@ -1,21 +1,64 @@
+'use client'
+import Image from "next/image"
 import Link from "next/link"
+import { usePathname } from "next/navigation"
+import { useState } from "react"
+import logo_symbol from "../../../assets/images/logo/logo_symbol.svg"
 import { routes } from "../../../services/routes/routes"
+import { NavLink } from "../../../types/routes"
+import { Button } from "../../Button/Button"
+import { IconButton } from "../../IconButton/IconButton"
+import { NavbarLink } from "../NavbarLink/NavbarLink"
 
 /**
  * Display Navbar with Logo and Links.
  * @example <GlobalNavbar />
  */
 export const GlobalNavbar: React.FC = () => {
-    return <nav>
-        <div className="container flex my-5">
-            <div className="flex items-center flex-shrink-0 mr-6">
-                <Link href={routes.index} className="font-semibold">Beyond Life</Link>
+    const pathname = usePathname()
+
+    // Local States
+    const [isNavMobileOpen, setIsNavMobileOpen] = useState<boolean>(false)
+
+    const navLinks: NavLink[] = [
+        {
+            to: routes.index,
+            text: "Discover",
+        },
+        {
+            to: routes.lastWill.index,
+            text: "Testament",
+        }
+    ]
+
+    return <div className="bg-yellow-400 py-3 mb-5">
+        <nav className="container md:flex">
+            <div className="flex items-center justify-between md:mr-5">
+                {/* Logo */}
+                <Link className="flex items-center" href={routes.index}>
+                    <Image src={logo_symbol} alt="Logo" className="min-w-[25px] w-[25px] mr-2" />
+                    <h6 className="font-bold text-lg">BeyondLife</h6>
+                </Link>
+
+                {/* Mobile menu button */}
+                <IconButton className="block md:hidden" aria-label="Open mobile navigation" onClick={() => setIsNavMobileOpen((isNavMobileOpen) => !isNavMobileOpen)} icon="menu" />
             </div>
 
-            <ul className="flex gap-3 items-center">
-                <li><Link href={routes.lastWill.start}>Testament erstellen</Link></li>
-            </ul>
-        </div>
-    </nav>
+            {/* Navlinks */}
+            <ul className={`${isNavMobileOpen ? "block" : "hidden"} md:flex items-center gap-4 w-full py-5 md:py-0`}>
+                {/* Navlinks Start */}
+                {navLinks.map(navLink => <li key={navLink.text} className="mb-3 md:mb-0">
+                    <NavbarLink isActive={"to" in navLink && navLink.to === pathname} {...navLink}>{navLink.text}</NavbarLink>
+                </li>)}
 
+                {/* Navlinks End */}
+                <li className="ml-auto mb-3 md:mb-0">
+                    <Button kind="secondary" to={routes.account.login}>Login</Button>
+                </li>
+                <li>
+                    <NavbarLink to={routes.account.register} isActive={routes.account.register === pathname}>Register</NavbarLink>
```

Comment:
Wird da noch später was am Design angepasst? Finde, so sieht es doch irgendwie komisch aus, sollte der Register Button nicht vielleicht auch in die Mitte oder so? 
![image](https://github.com/dopeshot/beyond-life/assets/87226274/5728a205-adc9-4b92-9c2e-03139a434ec2)